src/platforms/unity/common/unity-lite/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Unity Lite
+description: "Unity Lite Sentry SDK."
+sidebar_order: 5
+---
+
+<PlatformContent includePath="getting-started-primer" />
+
+On this page, we get you up and running with Sentry's SDK, so that it will automatically report errors and exceptions in your application.
```

Comment:
We need to tell users that this is a specific SDK, with limited features. And that the sections on the menu on the left (particularly the _enriching events_ content https://sentry-docs-git-fork-semuserable-docs-unity.sentry.dev/platforms/unity/enriching-events/ ) don't necessarily apply to this SDK. For example. there's no `SentrySdk.ConfigureScope` in this SDK. Or `SentrySdk.AddBreadcrumb`. There's not even `SentrySdk.CaptureException`.